arwen/contexts/runtime.go

```diff
@@ -200,8 +200,7 @@ func (context *runtimeContext) makeInstanceFromContractByteCode(contract []byte,
 
 	context.saveCompiledCode(codeHash)
 
-	idContext := arwen.AddHostContext(context.host)
-	context.instance.SetContextData(idContext)
+	context.instance.SetContextData(uintptr(unsafe.Pointer(&context.host)))
```

Comment:
Extract pointer of `context.host` as a variable, for readability:
```suggestion
        hostReference := uintptr(unsafe.Pointer(&context.host))
	context.instance.SetContextData(hostReference)
```